pkg/scripts/testdata/TestKubeadmAmazonLinux-force.golden

```diff
@@ -88,7 +88,7 @@ cat <<EOF | sudo tee /etc/docker/daemon.json
 	"storage-driver": "overlay2",
 	"log-driver": "json-file",
 	"log-opts": {
-		"max-size": "100m"
+		"max-size": ""
```

Comment:
Can you add the LoggingConfig to the function below?

https://github.com/kubermatic/kubeone/blob/5f11decd23b06d9915af412be5f17fb0d3955467/pkg/scripts/os_test.go#L91-L111